### DISCUSSION TITLE
3d Onera Wing Optimization

### DISCUSSION THREAD

**Discussion by oguzkaann** [mdolab/dafoam#266] [Tutorials]
Hello sir,
ı am working on 3d optimization on DaFOAM. every time ı see this error. 

https://user-images.githubusercontent.com/95033071/154668939-39fe4ccc-1259-4bea-a619-432a0022dadf.png

Could you help me? What can i do?
thank.
Best regards.

**Comment by friedenhe** [comment]
It is likely that you do not have enough memory. Check this: https://dafoam.github.io/mydoc_get_started_faq.html#how-many-cpu-cores-to-use-and-how-much-memory-does-it-need
